

fs/reiser4/coord.c: In function `coords_equal':
fs/reiser4/coord.c:598: error: duplicate case value
fs/reiser4/coord.c:598: error: previously used here


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/reiser4/coord.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN fs/reiser4/coord.c~reiser4-sparc64-build-fix fs/reiser4/coord.c
--- 25/fs/reiser4/coord.c~reiser4-sparc64-build-fix	2004-08-18 23:15:34.509811208 -0700
+++ 25-akpm/fs/reiser4/coord.c	2004-08-18 23:15:43.348467528 -0700
@@ -594,6 +594,7 @@ coords_equal(const coord_t * c1, const c
 	assert("nikita-2840", c1 != NULL);
 	assert("nikita-2841", c2 != NULL);
 
+#if 0
 	/* assertion to track changes in coord_t */
 	cassert(sizeof(*c1) == sizeof(c1->node) +
 		sizeof(c1->item_pos) +
@@ -603,6 +604,7 @@ coords_equal(const coord_t * c1, const c
 		sizeof(c1->pad) +
 		sizeof(c1->offset) +
 		DEBUG_COORD_FIELDS);
+#endif
 	return
 		c1->node == c2->node &&
 		c1->item_pos == c2->item_pos &&
_
